spec/features/admin/non_admin_user_cannot_create_user_as_admin_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+feature "Non admin user cannot create user as admin" do
```

Comment:
This spec is testing access rather than creation and again should have been covered in a controller spec.

Reply:
Done in sha: 33f2dfc